src/libs/MeadeStreamReader/MeadeStreamReader.cpp

```diff
@@ -3,30 +3,34 @@
 //
 
 #include "MeadeStreamReader.hpp"
+#include "FastPin.h"
 
 MeadeStreamReader::MeadeStreamReader()
 {
-
 }
 
-MeadeStreamReader::MeadeStreamReader(Stream *stream, String(*callback)(String)) {
+MeadeStreamReader::MeadeStreamReader(Stream *stream, String (*callback)(String))
+{
     init(stream, callback);
+    FastPin<7>::output();
 }
 
 void MeadeStreamReader::init(Stream *stream, String (*callback)(String))
 {
-    this->stream = stream;
+    this->stream   = stream;
     this->callback = callback;
-    this->newData = false;
-    this->i = 0;
+    this->newData  = false;
+    this->i        = 0;
 }
 
 
-void MeadeStreamReader::loop() {
-    if (stream) // TODO error on not init?
+void MeadeStreamReader::loop()
+{
+    FastPin<7>::high();
```

Comment:
I assume this is for debugging with a logic analyzer? It should be in #ifdef DEBUG_SOMETHING unless you're planning to rip it all out when you're done.

Reply:
Yeah. I'll be pulling this out before merge.